frontend/components/bc/input/BcInputCheckbox.vue

```diff
@@ -27,14 +27,15 @@
       {{ label }}
     </label>
     <BcTooltip 
-      v-if="infoText"
+      v-if="infoText || $slots.tooltip"
       class="bc-input-checkbox__info"
-      tooltip-width="217px"
+      tooltip-width="220px"
       tooltip-text-align="left"
     >
       <FontAwesomeIcon :icon="faInfoCircle" />
       <template #tooltip>
         {{ infoText }}
+        <slot v-if="$slots.tooltip" name="tooltip" />
```

Comment:
don't think the `v-if="$slots.tooltip" ` if is necessary ... if $slots.tooltip is not set then the slot will not be added.

Reply:
true thx